.changeset/happy-mayflies-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sap-cloud-sdk/openapi-generator": minor
+---
+
+OpenApi clients are generated with `in: header` parameters defined in the spec.
```

Comment:
[req] Use imperative: aka. Generate OpenAPI clients... (also OpenAPI not OpenApi)